platform/security/core/security-core-impl/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java

```diff
@@ -112,16 +116,24 @@ public LogoutRequest extractSamlLogoutRequest(@NotNull String samlLogoutResponse
    *
    * @param nameIdString NameId of user to log out
    * @param issuerOrEntityId The Issuer of the LogoutRequest
+   * @param sessionIndexes
    * @return the built <code>LogoutRequest</code>
    */
   @Override
   public LogoutRequest buildLogoutRequest(
-      @NotNull String nameIdString, @NotNull String issuerOrEntityId) {
-    return buildLogoutRequest(nameIdString, issuerOrEntityId, UUID.randomUUID().toString());
+      @NotNull String nameIdString, @NotNull String issuerOrEntityId, List<String> sessionIndexes) {
+    return buildLogoutRequest(nameIdString, issuerOrEntityId, generateUuid(), sessionIndexes);
+  }
+
+  private String generateUuid() {
+    return getUuidGenerator().generateUuid();
```

Comment:
❓ Should we just use the member variable here instead and get rid of the getter?

Reply:
Yes. I pulled it into a method to make it a little easier to mock. But it looks like I eventually removed that test because I don't find any other callers of that method.